x-pack/plugins/ml/public/application/components/anomalies_table/links_menu.js

```diff
@@ -411,6 +456,22 @@ class LinksMenuUI extends Component {
           </EuiContextMenuItem>
         );
       });
+    } else {
+      items.push(
+        <EuiContextMenuItem
+          key={`auto_raw_data_url`}
+          icon="popout"
+          onClick={() => {
+            this.closePopover();
+            this.openInDiscover();
+          }}
+          data-test-subj={`mlAnomaliesListRowAction_rawDataButton`}
+        >
+          {i18n.translate('xpack.ml.anomaliesTable.linksMenu.rawData', {
```

Comment:
For react components, `FormattedMessage` is preferred according to the Kibana guidelines

Reply:
Updated here https://github.com/elastic/kibana/pull/120450/commits/d6e4b9adae89b65416ff57ac59a43136b49e508b